fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -223,12 +217,20 @@ DOUBLE_VALUE
     | '.' DIGIT+ EXPONENT
     ;
 
+SINGLE_QUOTED_TEXT
+    : '\'' ( ~'\'' | '\'\'' )* '\''
+    ;
+
+DOUBLE_QUOTED_TEXT
+    : '"' ( '\\'. | '""' | ~('"'| '\\') )* '"'
```

Comment:
`'""'` why allow this？

`""""` is two double quoted texts or one?

Reply:
"" is a valid string
"""", because the internal "" is not escaped, so it is actually two strings, but because the two strings are directly connected, it is not grammatical, and an error will be reported

Reply:
"" is valid string
"" inside "", will be escaped as single ", which is compatible with mysql syntax. In fact, """", the output is a string "